bin/tronview

```diff
@@ -60,6 +60,14 @@ def parse_cli():
         help="Solely displays stderr",
         default=0,
     )
+    parser.add_argument(
+        "--events",
+        "-E",
+        action="count",
```

Comment:
Are you planning on adding a count argument to the events GET endpoint later?

Reply:
no, i  just copied the definition from other args, this should really be store_true or something.